playbooks/roles/filebeat/templates/filebeat.yml.j2

```diff
@@ -0,0 +1,215 @@
+filebeat:
+  prospectors:
+    -
+      paths:
+        - "{{ nginx_log_dir }}/kibana.access.log"
+      fields:
+        nginx_site: kibana
+        openedx_log_format: nginx_access
+    -
+      paths:
+        - "{{ nginx_log_dir }}/carnegie_access.log"
+        - "{{ nginx_log_dir }}/convenience_access.log"
+        - "{{ nginx_log_dir }}/verify_access.log"
+        - "{{ nginx_log_dir }}/access.log"
+        - "{{ nginx_log_dir }}/access_verify.log"
+      fields:
+        nginx_site: other
+        openedx_log_format: nginx_access
+    -
+      paths:
+        - "{{ nginx_log_dir }}/cms_access.log"
+      fields:
+        nginx_site: cms
+        openedx_log_format: nginx_access
+    -
+      paths:
+        - "{{ nginx_log_dir }}/lms_access.log"
+      fields:
+        nginx_site: lms
+        openedx_log_format: nginx_access
+    -
+      paths:
+        - "{{ nginx_log_dir }}/carnegie_error.log"
+        - "{{ nginx_log_dir }}/convenience_error.log"
+        - "{{ nginx_log_dir }}/error.log"
+        - "{{ nginx_log_dir }}/error-verify.log"
+        - "{{ nginx_log_dir }}/verify_error.log"
+      fields:
+        nginx_site: other
+        openedx_log_format: nginx_error
+    -
+      paths:
+        - "{{ nginx_log_dir }}/cms_error.log"
+      fields:
+        nginx_site: cms
+        openedx_log_format: nginx_error
+    -
+      paths:
+        - "{{ nginx_log_dir }}/lms_error.log"
+      fields:
+        nginx_site: lms
+        openedx_log_format: nginx_error
+    -
+      paths:
+        - "{{ COMMON_LOG_DIR }}/cms/edx.log"
+      fields:
+        service_variant: cms
+        openedx_log_format: openedx_standard
+    -
+      paths:
+        - "{{ COMMON_LOG_DIR }}/lms/edx.log"
+      fields:
+        service_variant: lms
+        openedx_log_format: openedx_standard
+    -
+      paths:
+        - "{{ COMMON_LOG_DIR }}/notifier/edx.log"
+      fields:
+        service_variant: notifier
+        openedx_log_format: openedx_standard
+    -
+      paths:
+        - "{{ COMMON_LOG_DIR }}/xqueue/edx.log"
+      fields:
+        service_variant: xqueue
+        openedx_log_format: openedx_standard
+    -
+      paths:
+        - "{{ COMMON_LOG_DIR }}/certs/edx.log"
+      fields:
+        service_variant: certs
```

Comment:
As you squash all of the similiar-format syslog messages, double-check that service_variant is provided.